tests/integration_tests/mock_llm_outputs.py

```diff
@@ -9,11 +9,14 @@ def openai_completion_create(prompt, *args, **kwargs):
         pydantic.COMPILED_PROMPT: pydantic.LLM_OUTPUT,
         pydantic.COMPILED_PROMPT_REASK_1: pydantic.LLM_OUTPUT_REASK_1,
         pydantic.COMPILED_PROMPT_REASK_2: pydantic.LLM_OUTPUT_REASK_2,
+        string.COMPILED_PROMPT: string.LLM_OUTPUT,
+        string.COMPILED_PROMPT_REASK: string.LLM_OUTPUT_REASK,
     }
 
     try:
         return mock_llm_responses[prompt]
     except KeyError:
+        print(prompt)
```

Comment:
nit: Was this meant to be left in?

Reply:
Yes, because it shows the failed output in stdout. Otherwise I would rerun it with a breakpoint on that line, just to see what the prompt is.